src/transformers/models/albert/modeling_albert.py

```diff
@@ -358,6 +361,66 @@ def forward(
         return (layernormed_context_layer, attention_probs) if output_attentions else (layernormed_context_layer,)
 
 
+class AlbertSdpaAttention(AlbertAttention):
+    def __init__(self, config):
+        super().__init__(config)
+        self.dropout_prob = config.attention_probs_dropout_prob
+        self.require_contiguous_qkv = version.parse(get_torch_version()) < version.parse("2.2.0")
```

Comment:
You can use a defined variable instead
```
from ...pytorch_utils import is_torch_greater_or_equal_than_2_2
```

Reply:
This wasn't resolved

Reply:
Did I miss something? I think I added this.

Reply:
My bad - I must have been looking at an older diff. All good!